charts/fission-core/templates/deployment.yaml

```diff
@@ -221,6 +221,14 @@ spec:
           value: "{{ .Values.pullPolicy }}"
         - name: ENABLE_ISTIO
           value: "{{ .Values.enableIstio }}"
+        - name: FETCHER_RESOURCE_REQUESTS_CPU
+          value: "{{ .Values.fetcher.requestsCPU }}"
```

Comment:
One more thing I missed earlier `{{ default "10m" .Values.fetcher. requests CPU }}`  so that even if the user does not specify (By removing from values.yaml) - then defaults are in place.